frontstage/templates/sign-in/sign-in.account-locked.html

```diff
@@ -0,0 +1,37 @@
+{% import 'partials/section.html' as section %}
+{% extends "layouts/_twocol.html" %}
+
+{% block page_title %}Trouble signing in? - ONS Business Surveys{% endblock %}
+
+{% block main %}
+    <h1 class="saturn">Check your email</h1>
+
+    <p>You've tried to sign in few times with the wrong details.</p>
+    <p>If your email is registered to an account, we will send an email explaining how to sign in.</p>
+    <p>If you do not receive this email in 15 minutes, you can try the following: </p>
+
+    <p>
+        <li>Check your spam folder</li>
+        <li>Check that you have entered your email address correctly</li>
+        <li>Ensure that you have registered an account using this email address</li>
+    </p>
+    <p>If you're still having problems, call us on <a href="tel:0300 1234 931">0300 1234 931</a> </p>
+    <p>or send an email to <a href="mailto:surveys@ons.gov.uk">surveys@ons.gov.uk</a></p>
+
+    <div class="details js-details" data-guidance-label="Help with email" data-guidance="Help with email" data-hide-label="Help with email" data-show-label="Help with email">
+
+      <a class="details__link js-details-trigger js-details-label mars" id="guidance-help-with-email" data-guidance-trigger="true" aria-expanded="true" href="#guidance-help-with-email" aria-controls="guidance-help-with-email-body">Help with email</a>
+
+      <div class="details__main js-details-body" id="guidance-help-with-email-body">
+        <div class="details__content mars">
+          <div>
+            <p>If you do not receive this email in 15 minutes, you can try the following: </p>
+              <li>Check your spam folder</li>
+              <li>Check that you've entered your email address correctly </li>
+              <li>Ensure that you've registered an account using this email address </li>
+              <br>If you're still having problems, call us on 0300 1234 931 or send an <br/>email to <a href="mailto:surveys@ons.gov.uk">surveys@ons.gov.uk</a>.</p>
```

Comment:
Nothing major but should the phone no. be a hyperlink to match the non JS version.